flocker/node/_docker.py

```diff
@@ -29,6 +30,9 @@
     RestartNever, RestartAlways, RestartOnFailure, pset_field, pvector_field)
 
 
+logger = logging.getLogger(__name__)
```

Comment:
I don't think we've hooked the stdlib logging system up to anything - so these log messages don't go anywhere. :/  I double-checked on buildbot and I didn't see the messages anywhere in `test.log` or on stdout.

`eliot.Message.new(keyword=arguments).write()` is probably the easiest thing to do that will actually send the logs somewhere.  `with eliot.start_action(action_type=stuff, keyword=arguments):` is another pretty easy thing to do if you want parent/child relationships or success/failure results in the logs.

Like the `logger`-based logging you added, this stuff doesn't need test coverage right now.